Makefile

```diff
@@ -1,31 +1,38 @@
-TARGETS = naive cuda cupla-cuda-async cupla-seq-seq-async cupla-seq-seq-sync cupla-tbb-seq-async cupla-omp2-seq-async kokkos-serial kokkos-openmp kokkos-cuda
+TARGETS = naive alpakaSer alpakaTBB alpakaGpuCuda cuda cupla-cuda-async cupla-seq-seq-async cupla-seq-seq-sync cupla-tbb-seq-async cupla-omp2-seq-async kokkos-serial kokkos-openmp kokkos-cuda
```

Comment:
Could these be renamed to `alpaka-ser alpaka-tbb alpaka-gpu-cuda` (or the last one just `alpaka-cuda`) to follow the naming convention?

Reply:
Renamed in #13 !